plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeColumnHandle.java

```diff
@@ -161,4 +161,9 @@ public static DeltaLakeColumnHandle mergeRowIdColumnHandle()
     {
         return new DeltaLakeColumnHandle(ROW_ID_COLUMN_NAME, MERGE_ROW_ID_TYPE, OptionalInt.empty(), ROW_ID_COLUMN_NAME, MERGE_ROW_ID_TYPE, SYNTHESIZED, Optional.empty());
     }
+
+    public static boolean isMetadataColumnHandle(DeltaLakeColumnHandle columnHandle)
```

Comment:
We will probably forget updating this method in the future. Can we add a code comment (e.g. Update isMetadataColumnHandle method when adding a new metadata columns) above `PATH_COLUMN_NAME` constant? Not perfect solution but better than nothing.